components/home/TestimonialsSection.vue

```diff
@@ -94,27 +93,19 @@
 </template>
 
 <script setup lang="ts">
-// todo: types declaration for hooper
-import { Hooper, Slide, Pagination as HooperPagination } from 'hooper'
+import Flicking from '@egjs/vue3-flicking'
 import img1 from 'assets/media/img/1.png'
 import img2 from 'assets/media/img/2.png'
 import img3 from 'assets/media/img/3.png'
 import img4 from 'assets/media/img/4.png'
 import img5 from 'assets/media/img/5.png'
 import img6 from 'assets/media/img/6.png'
+import { Pagination } from '@egjs/flicking-plugins'
 
-type Testimonial = {
-  name: string
-  title: string
-  testimony: string
-}
+const plugins = [new Pagination({ type: 'bullet' })]
 
-const props = defineProps({
-  testimonials: {
-    type: Array<Testimonial>,
-    required: true,
-  },
-})
+const props = defineProps(['testimonials'])
```

Comment:
Same. Not sure why prefer the flat syntax as it removed type inference and removed the default value of empty array.

Reply:
I was the one that changed it to the other syntax initially IIRC, and created my own types, which i thought were correct, but actually weren't. The props that are passed are actually nuxt content types, so they weren't actually compatible.